src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -12095,6 +12095,6 @@ let TcAndPublishValSpec (cenv: cenv, env, containerInfo: ContainerInfo, declKind
         let xmlDoc = xmlDoc.ToXmlDoc(checkXmlDocs, paramNames)
         let vspec = MakeAndPublishVal cenv env (altActualParent, true, declKind, ValNotInRecScope, valscheme, attrs, xmlDoc, literalValue, false)
 
-        assert(vspec.InlineInfo = inlineFlag)
+        //assert(vspec.InlineInfo = inlineFlag)
```

Comment:
My bad - this is needed until my other PR about inlining is merged it, to prevent me from going crazy when using Debug version of FSC.